spectator/registry.h

```diff
@@ -47,6 +47,7 @@ class Registry {
 
   std::vector<std::shared_ptr<Meter>> Meters() const noexcept;
   std::vector<Measurement> Measurements() const noexcept;
+  std::size_t Size() const noexcept { return meters_.size(); }
```

Comment:
Is this thread safe?

Reply:
It's not but I wasn't sure that potentially getting an incorrect size was important. You're right in calling it out since I can imagine this leading to difficult to debug situations, and especially because this is mostly for debugging and should have no effect on our performance. #64 fixes it.